### PR TITLE
Add SICD test images

### PR DESCRIPTION
**Pull request by RanbirAulakh**
**Issue #, if available:** n/a

### Notes
- Add SICD test images to work with this PR: https://github.com/aws-solutions-library-samples/osml-model-runner-test/pull/5

### Testing
- n/a 

Before you submit a pull request, please make sure you have to following:

- [x] Your code contains tests that cover all new code and changes
- [x] All new and existing tests passed
- [x] I have read the [Contributing Guidelines](https://github.com/aws-solutions-library-samples/guidance-for-overhead-imagery-inference-on-aws/blob/main/CONTRIBUTING.md) and agree to follow the [Code of Conduct](
https://github.com/aws-solutions-library-samples/guidance-for-overhead-imagery-inference-on-aws/blob/main/CODE_OF_CONDUCT.md))

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by RanbirAulakh** [comment]
I will need to submit another commit for github actions as part of this.

File: .gitattributes
Comment (edparris) [inline .gitattributes:1]: No need to change this commit but consider moving to a pattern matching approach that will use Git LFS for all NITF images without needing to list them individually. The .gitattributes pattern matching is somewhat limited but something like this has been working so far in other packages:

`
*.[nN][iI][tT][fF] filter=lfs diff=lfs merge=lfs -text
*.[nN][tT][fF] filter=lfs diff=lfs merge=lfs -text
*.[gG][eE][oO][tT][iI][fF][fF] filter=lfs diff=lfs merge=lfs -text
*.[tT][iI][fF][fF] filter=lfs diff=lfs merge=lfs -text
*.[tT][iI][fF] filter=lfs diff=lfs merge=lfs -text
`

**Review by edparris — APPROVED** [review]
No significant issues; just comments to consider for future work.

File: assets/images/README.md [hunk 1]
```diff
@@ -22,3 +22,9 @@ functionality, along with performance when handling tiling a heavier image.
 ## meta.ntf
 This is a small NITF image with classification metadata embedded in it. It is used to test model runners ability to
 parse that metadata and embedded in the corresponding FeatureCollection output properties.
+
+## sicd-capella-chip.ntf / sicd-umbra-chip
+This is a small chip sicd-capella/sicd-umbra image which is taken from large capella/umbra imagery. It is used to test the model runners ability to parse that metadata and embedded in the corresponding FeatureCollection output properties. These images were taken from: [capella](https://radiantearth.github.io/stac-browser/#/external/capella-open-data.s3.us-west-2.amazonaws.com/stac/capella-open-data-by-datetime/capella-open-data-2021/capella-open-data-2021-8/capella-open-data-2021-8-7/CAPELLA_C03_SP_SICD_HH_20210807095836_20210807095839/CAPELLA_C03_SP_SICD_HH_20210807095836_20210807095839.json?.language=en&.asset=asset-HH) and [umbra](https://umbra.space/open-data)
+
+## sicd-interferometric-hh.nitf
```
Comment (drduhe) [inline assets/images/README.md:29]: Can we include the pixel dims and the size of the file in this README? I see that it's also not there for meta.ntf if you want to add that one too.

File: assets/images/README.md [hunk 2]
```diff
@@ -22,3 +22,9 @@ functionality, along with performance when handling tiling a heavier image.
...
+This is a larger Nitf image which is taken from this website: https://github.com/ngageoint/six-library/wiki/Sample-SICDs, this is to ensure that the OSML can handle a large SAR data.
```
Comment (drduhe) [inline assets/images/README.md:30]: Same

File: .gitattributes
Comment (drduhe) [reply .gitattributes:1]: I would like using this across all our packages actually. +1

**Review by drduhe — CHANGES_REQUESTED** [review]

File: .gitattributes
Comment (RanbirAulakh) [reply .gitattributes:1]: Fixed in next commit.

File: assets/images/README.md (hunk 1 above)
Comment (RanbirAulakh) [reply assets/images/README.md:29]: Fixed in next commit.

File: assets/images/README.md (hunk 2 above)
Comment (RanbirAulakh) [reply assets/images/README.md]: Fixed in next commit.

**Review by drduhe — APPROVED** [review]
LGTM

**Pull request merged**